File: output/golang/struct/DisplayInformation.go
package main

type DisplayInformation struct {
	Orientation string `json:"orientation" bson:"orientation"`
	Top int `json:"top" bson:"top"`
	Left int `json:"left" bson:"left"`
	Bottom int `json:"bottom" bson:"bottom"`
	Right int `json:"right" bson:"right"`
	Display_name string `json:"display_name" bson:"display_name"`
}

File: output/kotlin/models/overridden/DisplayInformation.kt
package overridden

interface IDisplayInformation {
    val orientation: String?
    val top: Int?
    val left: Int?
    val bottom: Int?
    val right: Int?
    val displayName: String?
}

data class DisplayInformation(
    override val orientation: String?,
    override val top: Int?,
    override val left: Int?,
    override val bottom: Int?,
    override val right: Int?,
    override val displayName: String?,
) : IDisplayInformation